Helper/ProductData.php

```diff
@@ -184,6 +185,18 @@ public function tagsUseCategories($storeId = null)
         return boolval($this->getSettingsVal(self::XML_CONTENT_USE_CATEGORIES, $storeId));
     }
 
+    /**
+     * Is remove product in sailthru
+     *
+     * @param string|null $storeId
+     *
+     * @return bool
+     */
+    public function isRemoveInSailthru($storeId = null)
```

Comment:
- Rename `isRemoveInSailthru` to `shouldRemoveContentInSailthru`
- INstead of bootval() can you simply use type casting via `(bool)`?